api.py

```diff
@@ -0,0 +1,194 @@
+import logging
+from typing import Optional
+from uuid import uuid4
+from fastapi import BackgroundTasks, FastAPI, HTTPException, status, Response
+from run_pipeline import run
+from enum import Enum
+from pydantic import BaseModel
+
+
+logger = logging.getLogger(__name__)
+api = FastAPI()
+
+
+class Status(Enum):
+    CREATED = "CREATED"
+    PROCESSING = "PROCESSING"
+    DONE = "DONE"
+    ERROR = "ERROR"
+
+
+StatusToHTTP = {
+    Status.CREATED: status.HTTP_201_CREATED,
+    Status.PROCESSING: status.HTTP_202_ACCEPTED,
+    Status.DONE: status.HTTP_200_OK,
+    Status.ERROR: status.HTTP_500_INTERNAL_SERVER_ERROR,
+}
+
+
+class Task(BaseModel):
+    input_uri: str
+    output_uri: str
```

Comment:
I don't get this feedback. Can you explain/reformulate?

Reply:
Technically, `output_uri` is not a uri: it doesn't point to a single object. 
Instead, `output_uri` is the destination _folder_ in S3 where both the output audio and provenance will be uploaded to.
And technically in S3, a 'folder' does not exist: it is just a _prefix_ that contains `/` characters.

Reply:
And the provenance filename (as well as the audio filename, for that matter) are configured, the caller (from Airflow, in this case) can only guess the actual uris of the output objects (from the prefix). It would be nice to make them explicit.